_events/sr2020/virtual-competition.md

```diff
@@ -1,22 +0,0 @@
----
```

Comment:
Is there a way we can make this page redirect to somewhere suitable?
I think we have a bunch of things which link to this page. It would be a shame to break those links.

Reply:
Is there a built-in way to do this?

Reply:
I'm not completely sure. I think we do have redirects somewhere in the website.

Reply:
I changed my mind and added a table of links instead